unit_tests/IssueUnitTests.d

```diff
@@ -106,3 +105,43 @@ struct property_t
 D");
 
 }
+
+// Fix 10: Embedded struct not generated.
+unittest
```

Comment:
This should follow the same structure as in #71. A minimal unit test for structs and the full test case for the issue in a separate file.